### PR TITLE
ADEN-11088 fixed viewability for UAP

### PR DESCRIPTION
**Pull request by preludek**

### CODE REVIEW

**Comment by preludek** [comment]
Changes here also affect the old UAP (transition state). This was working correctly imo only because old UAP is always in the viewport at first (either top of page or immediately get's sticked).

Also I removed `shouldStickUntillSlotViewed` since in design it should always stay sticky until viewability is counted.

File: platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts [hunk 1]
```diff
@@ -39,13 +42,17 @@ export class SlotDecisionEmbeddedBigToEmbeddedResolvedHandler implements Templat
 	private async staticTransitionHandler(
 		transition: TemplateTransition<'embeddedResolved'>,
 	): Promise<void> {
-		this.timeout
-			.isViewedAndDelayed()
+		merge(this.getViewabilityStream())
```
Comment (Loczi94) [inline platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts:45]: I don't think we need `merge` here.

File: platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts [hunk 2]
```diff
@@ -39,13 +42,17 @@ export class SlotDecisionEmbeddedBigToEmbeddedResolvedHandler implements Templat
...
 			.pipe(
-				filter((viewedAndDelayed) => !this.isVideo && viewedAndDelayed),
+				filter(() => !this.isVideo),
```
Comment (Loczi94) [inline platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts:47]: I don't like the idea of using private variable to switch between these two streams but I couldn't figure out how to implement it using only RxJs operators.

Comment (preludek) [reply platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts:47]: Yeah, me neither 🙈  couldn't come up with anything better 🤷

File: platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts (hunk 1 above)
Comment (preludek) [reply platforms/shared/templates/handlers/slot/slot-decision-embedded-big-to-embedded-resolved-handler.ts]: Yup, fixed!

**Review by Loczi94 — APPROVED** [review]

**Pull request merged**